tasks/provision.yml

```diff
@@ -2,14 +2,14 @@
   assert:
     that: "{{ item }} is defined"
   with_items:
+    - mr_provisioner_machine_name
     - mr_provisioner_kernel_description
     - mr_provisioner_initrd_description
     - mr_provisioner_kernel_path
     - mr_provisioner_initrd_path
     - mr_provisioner_url
     - mr_provisioner_auth_token
     - mr_provisioner_preseed_name
-    - mr_provisioner_preseed_path
```

Comment:
Not this topic related? and it is required by the preseed module.

Reply:
My bad.